### PR TITLE
[RIP-20] Liquibase migration

### PR DESCRIPTION
**Pull request by Millrocious**
# Description
made first liquibase migration

Change type: #minor

- create all tables
- add default insertion to locale

### CODE REVIEW

**Comment by eduard-romaniuk** [comment]
@Millrocious please fill PR description properly

File: src/main/resources/db/changelog/changeset/create-start-tables.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     helicopters INTEGER NOT NULL,
+     vehicles_fuel_tanks INTEGER NOT NULL,
+     warships_cutters INTEGER NOT NULL,
+     cruise_missiles INTEGER NOT NULL,
+     uav_systems INTEGER NOT NULL,
+     special_military_equip INTEGER NOT NULL,
+     atgm_srbm_systems INTEGER NOT NULL
+);
+
+-- changeset Millrocious:create-table-fund
```
Comment (eduard-romaniuk) [inline src/main/resources/db/changelog/changeset/create-start-tables.sql]: One changeset per one sql file. Please leave only one changeset annotation on top.
For example: `-- changeset Millrocious:init-database`

File: src/main/resources/db/changelog/changeset/create-start-tables.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+CREATE TABLE IF NOT EXISTS fund(
+   id BIGSERIAL NOT NULL PRIMARY KEY,
+   name varchar(255) NOT NULL,
```
Comment (eduard-romaniuk) [inline src/main/resources/db/changelog/changeset/create-start-tables.sql]: Would be great to check that at least name unique

File: src/main/resources/db/changelog/changeset/create-start-tables.sql [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+   description text NOT NULL,
+   link varchar(255) NOT NULL,
+   update_date timestamp NOT NULL,
+   create_date timestamp NOT NULL
+);
+
+-- changeset Millrocious:create-table-locale
+CREATE TABLE IF NOT EXISTS locale(
+     id BIGSERIAL NOT NULL PRIMARY KEY,
+     name varchar(255) NOT NULL,
```
Comment (eduard-romaniuk) [inline src/main/resources/db/changelog/changeset/create-start-tables.sql]: Same here, name and iso_code should be unique

File: src/main/resources/db/changelog/changeset/create-start-tables.sql [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+     iso_code varchar(2) NOT NULL
+);
+
+-- changeset Millrocious:create-table-user
+CREATE TABLE IF NOT EXISTS "user"(
+     id BIGSERIAL NOT NULL PRIMARY KEY,
+     name varchar(255) NOT NULL,
+     email varchar(255) NOT NULL,
```
Comment (eduard-romaniuk) [inline src/main/resources/db/changelog/changeset/create-start-tables.sql]: Should be unique

**Review by eduard-romaniuk — CHANGES_REQUESTED** [review]

**Comment by eduard-romaniuk** [comment]
Also, please add default en and ua locales inserting in your migration

File: src/main/resources/db/changelog/changeset/create-start-tables.sql (hunk 1 above)
Comment (Millrocious) [reply src/main/resources/db/changelog/changeset/create-start-tables.sql]: fixed

File: src/main/resources/db/changelog/changeset/create-start-tables.sql (hunk 2 above)
Comment (Millrocious) [reply src/main/resources/db/changelog/changeset/create-start-tables.sql]: fixed

File: src/main/resources/db/changelog/changeset/create-start-tables.sql (hunk 3 above)
Comment (Millrocious) [reply src/main/resources/db/changelog/changeset/create-start-tables.sql]: fixed

File: src/main/resources/db/changelog/changeset/create-start-tables.sql (hunk 4 above)
Comment (Millrocious) [reply src/main/resources/db/changelog/changeset/create-start-tables.sql]: fixed

**Comment by eduard-romaniuk** [comment]
@Millrocious 
> Also, please add default en and ua locales inserting in your migration

File: src/main/resources/db/changelog/changeset/create-start-tables.sql [hunk 5]
```diff
@@ -0,0 +1,48 @@
+-- liquibase formatted sql
+-- changeset Millrocious:init-database
+CREATE TABLE IF NOT EXISTS statistic_data(
+     id BIGSERIAL NOT NULL PRIMARY KEY,
+     war_date DATE NOT NULL,
+     day_number INTEGER NOT NULL,
+     personal_units INTEGER NOT NULL,
```
Comment (eduard-romaniuk) [inline src/main/resources/db/changelog/changeset/create-start-tables.sql]: typo

Comment (Millrocious) [reply src/main/resources/db/changelog/changeset/create-start-tables.sql]: fixed

**Review by eduard-romaniuk — APPROVED** [review]

**Pull request merged**